src/Kovan-DssSpell.sol

```diff
@@ -16,39 +16,284 @@
+        });
+
+        CentrifugeCollateralValues memory RWA006 = CentrifugeCollateralValues({
+            MCD_JOIN: 0x039B74bD0Adc35046B67E88509900D41b9D95430,
+            GEM: 0x4E65F06574F1630B4fF756C898Fe02f276D53E86,
+            OPERATOR: 0x652A3B3b91459504A8D1d785B0c923A34D638218,
+            INPUT_CONDUIT: 0x652A3B3b91459504A8D1d785B0c923A34D638218,
+            OUTPUT_CONDUIT: 0x652A3B3b91459504A8D1d785B0c923A34D638218,
+            URN:  0x6fa6F9C11f5F129f6ECA4B391D9d32038A9666cD,
+            LIQ: 0x2881c5dF65A8D81e38f7636122aFb456514804CC,
+            gemID: "RWA006",
+            joinID: "MCD_JOIN_RWA006_A",
+            urnID: "RWA006_A_URN",
+            inputConduitID: "RWA006_A_INPUT_CONDUIT",
+            outputConduitID: "RWA006_A_OUTPUT_CONDUIT",
+            pipID: "PIP_RWA006",
+            ilk: "RWA006-A",
+            ilkRegistryName: "RWA006-A: Centrifuge: Peoples Company",
+            RATE: TWO_PCT,
+            CEIL: 20 * MILLION,
+            PRICE: 20_808_000 * WAD,
+            MAT: 10_000,
+            TAU: 0,
+            DOC: ""
+        });
+
+        CentrifugeCollateralValues[4] memory collaterals = [RWA003, RWA004, RWA005, RWA006];
+
+        // integrate RWA003-006
+        for (uint i = 0; i < collaterals.length; i++) {
+            integrateCentrifugeCollateral(collaterals[i]);
+        }
+
+        // Increase debt ceiling of RWA002 from 5M to 20M
+        bytes32 ilk = bytes32("RWA002-A");
+        address MIP21_LIQUIDATION_ORACLE = DssExecLib.getChangelogAddress("MIP21_LIQUIDATION_ORACLE");
+
+        DssExecLib.increaseIlkDebtCeiling(ilk, 15 * MILLION, true);
+
+        // [ (debt ceiling) + (2 years interest at current rate) ] * mat, i.e.
+        // 20MM * 1.035^2 * 1.05 as a WAD
+        Bumpable(MIP21_LIQUIDATION_ORACLE).bump(ilk, 22_495_725 * WAD);
+        DssExecLib.updateCollateralPrice(ilk);
+
+        // bump changelog version
+        DssExecLib.setChangelogVersion("1.1.x");
+    }
+
+    function integrateCentrifugeCollateral(CentrifugeCollateralValues memory collateral) internal {
+        address MIP21_LIQUIDATION_ORACLE =
+            DssExecLib.getChangelogAddress("MIP21_LIQUIDATION_ORACLE");
+
+        address vat = DssExecLib.vat();
+
+        // Sanity checks
+        require(GemJoinAbstract(collateral.MCD_JOIN).vat() == vat, "join-vat-not-match");
+        require(GemJoinAbstract(collateral.MCD_JOIN).ilk() == collateral.ilk, "join-ilk-not-match");
+        require(GemJoinAbstract(collateral.MCD_JOIN).gem() == collateral.GEM, "join-gem-not-match");
+        require(GemJoinAbstract(collateral.MCD_JOIN).dec() == DSTokenAbstract(collateral.GEM).decimals(), "join-dec-not-match");
+
+        RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).init(
+            collateral.ilk, collateral.PRICE, collateral.DOC, collateral.TAU
+        );
+        (,address pip,,) = RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).ilks(collateral.ilk);
+
+        // Set price feed for RWA003
```

Comment:
this should be "Set price feed for RWA-00x".

Reply:
Fixed!